src/modules/flow/components/NodeComponent.tsx

```
@@ -42,6 +44,20 @@ export const NodeComponent: React.FC<NodeComponentProps> = ({
   onShowResult,
   isBeingEdited = false,
 }) => {
+  const { t, i18n } = useTranslation();
+  const currentLanguage = i18n.language; // Get current language for node translations
+  
+  // Helper function to get translated fields from node's i18n
+  const getTranslatedFields = (node: NodeType, lang: string) => {
+    const i18nData = (node as any).i18n?.[lang] || (node as any).i18n?.en;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)